searchcore/src/vespa/searchcore/proton/matching/match_thread.cpp

```diff
@@ -414,12 +436,16 @@ MatchThread::run()
                         resultContext->sort->hasSortData(),
                         resultContext->grouping.get() != 0));
         get_token_timer.done();
+        trace->addEvent(5, "Start result processing");
         processResult(matchTools->getHardDoom(), std::move(result), *resultContext);
     }
     total_time.stop();
     total_time_s = total_time.elapsed().sec();
     thread_stats.active_time(total_time_s - wait_time_s).wait_time(wait_time_s);
+    trace->addEvent(4, "Start thread merge");
     mergeDirector.dualMerge(thread_id, *resultContext->result, resultContext->groupingSource);
+    trace->addEvent(4, "MatchThread::run Done");
+    trace->done();
```

Comment:
make sure start of thread trace has appropriate time tag to account for correct trace duration.

Reply:
Removed done() for consistency.